build/reqpacks/build

```diff
@@ -17,6 +16,15 @@ else
 	RGSYNC_VERSION=99.99.99
 fi
 
+if [[ -z $GEARS_VERSION ]]; then
+	# map rgsync versions to Gears versions
+	if [[ $RGSYNC_VERSION == 99.99.99 ]]; then
+		GEARS_VERSION=master
+	else
+		GEARS_VERSION=1.0.1
```

Comment:
Shouldn't this be `GEARS_VERSION=$RGSYNC_VERSION`?

Reply:
Are they necessarily aligned? They are in lockstep?

Reply:
I see what you mean, so we leave it hard coded for now?